tests/integration/test.replication.js

```diff
@@ -19,6 +19,16 @@ adapters.forEach(function (adapters) {
 
     var dbs = {};
 
+    before(function () {
+      // briefly increase the limit from 10 because we momentarily
+      // go over during these tests
+      PouchDB.setMaxListeners(20);
+    });
+
+    after(function () {
+      PouchDB.setMaxListeners(10);
+    });
+
```

Comment:
I probably should have broken `test.replication.js` into two separate tests and only had this `setMaxListeners` on one of them, but I got a little lazy here.